docs/changes.rst

```diff
@@ -265,6 +265,14 @@ Released: not yet
   the underlying connection. Used that function in the 'mof_compiler' script
   to remove a ResourceWarning about unclosed sockets. (issue #2610)
 
+* Test: Added support for end2end testing of WBEM servers based on
+  easy-server server and vault files. The server files can specify WBEM servers
```

Comment:
suggest change easy-server to easy-server python package

Reply:
DONE - changed to "server and vault files of the 'easy-server' Python package".